### PR TITLE
added java and python solution

### PR DESCRIPTION
**Pull request by hussain-2004**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
HAHA NO WAY SOMEONE SUBMITTED A SOL FOR THE SAME PROBLEM

**Comment by SansPapyrus683** [comment]
*ahem*

no yeah this one is a lot more consistent with the c++ version so

**Comment by hussain-2004** [comment]
Java Acceptance Link: https://codeforces.com/contest/581/submission/268305441
Python Acceptance Link : https://codeforces.com/contest/581/submission/268307127

@SansPapyrus683 , Please check the ac.

File: solutions/bronze/cf-581D.mdx [hunk 1]
```diff
@@ -89,4 +89,170 @@ int main() {
...
+
+</JavaSection>
+<PySection>
+
+```py
+def main():
+	import sys
+
+	input = sys.stdin.read
+	data = input().split()
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:186]:
```suggestion
	data = sys.stdin.read().split()
```

**Comment by hussain-2004** [comment]
@SansPapyrus683 , Made changes 

Python Acceptance Link : https://codeforces.com/contest/581/submission/268406970
pushed the changes.

File: solutions/bronze/cf-581D.mdx [hunk 2]
```diff
@@ -89,4 +89,169 @@ int main() {
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+import java.util.*;
+
+public class Main {
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-581D.mdx:97]: bruh why'd you undo my class name change to `ThreeLogos`

Comment (hussain-2004) [reply solutions/bronze/cf-581D.mdx:97]: So you want me to change the class name to ThreeLogos right?

**Comment by hussain-2004** [comment]
@SansPapyrus683 , Verify the changes

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by SansPapyrus683** [comment]
also no need to ping me lol i get emails regardless

**Pull request merged**